quickstart/README.md

```diff
@@ -49,6 +49,25 @@ To retrieve fruits:
 curl -X GET http://localhost:8080/fruits
 ```
 
+## Reactive demo
+
+This application provides a reactive version of REST controllers. 
+To see how they work, go to `http://localhost:8080/reactive-fruits.html`
+You can add fruits to the list via the form. This time the logic will be executed in a reactive way. 
+
+Alternatively, you can use curl commands to interact with the underlying REST API.
```

Comment:
This is a bit confusing, it's just a curl invocation, so you can't really tell how this was executed.

Reply:
I don't understand. What different would you want here?
I think that curl is enough. Before this section, there is a description of controller that uses Multi so users will know that this is using `Mutiny` underneath.